examples/meta/generator/targets/python.json

```diff
@@ -11,9 +11,33 @@
     "Comment": "#$comment\n",
     "Init": {
         "Construct": "$name = $type($arguments)",
-        "Copy": "$name = $expr"
+        "Copy": "$name = $expr",
+        "BoolVector": "$name = np.zeros( ($arguments), dtype='bool')",
+        "CharVector": "$name = np.zeros( ($arguments), dtype='|S1')",
+        "ByteVector": "$name = np.zeros( ($arguments), dtype='uint8')",
+        "WordVector": "$name = np.zeros( ($arguments), dtype='uint16')",
+        "ShortVector": "$name = np.zeros( ($arguments), dtype=???????)",
```

Comment:
I can't import some of these from the Python interface, so can't check what type they should have. Maybe they just need to be removed?